manager/src/main/java/com/redhat/service/bridge/manager/BridgesService.java

```diff
@@ -16,9 +17,31 @@
 
     Bridge getBridge(String id);
 
-    Bridge getBridge(String id, String customerId);
-
-    Bridge getReadyBridge(String bridgeId, String customerId);
+    /**
+     * Provides Bridge instance for given id/name. It throws `ItemNotFoundException` if bridge instance not found.
+     * 
+     * @param bridgeIdentifier It could be a bridge Id or bridge name.
+     * @param customerId Customer id.
+     * @return Bridge instance.
+     */
+    Bridge getBridgeByBridgeIdentifier(String bridgeIdentifier, String customerId);
```

Comment:
`getBridgeByBridgeIdentifier` sounds like a tongue-twister. What's wrong with `getBridgeById`?

Reply:
`BridgeIdentifier` could be an `Bridge Id` or a `Bridge Name` that why I change `getBridgeById` to `getBridgeByBridgeIdentifier` but if we suggest I could change this method name to `getBridgeByIdentifier`.

Reply:
Using `Id` and `Identifier` with two different logical meanings is very confusing, since in plain english the former is an abbreviation of the latter.

We should find another name (like `key` for example).

Reply:
As suggested by @robpblake, I changed it to `IdOrName`. Now function is `getBridgeByIdOrName`